security_routes/auth_routes.py

```diff
@@ -23,7 +23,7 @@
 
 router = APIRouter()
 
-oauth2_scheme = OAuth2PasswordBearer(tokenUrl="token")
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl="latest/token")
```

Comment:
What's the tokenUrl here? What does it do?

Reply:
That was the problem, since that route did not exist, and in the UI it was pointing to /token it got 404, in the tests and everything else we did have /latest/token. Maybe at some point a /token route existed, in case of v1 and v2 it works with an API key rather than the /latest/token route

Reply:
OK yep, makes sense. I think we can go ahead and merge this.